meta-lmp-base/recipes-containers/docker/files/docker.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Docker Application Container Engine
 Documentation=https://docs.docker.com
-After=network-online.target docker.socket firewalld.service containerd.service fio-docker-fsck.service
+After=network-online.target docker.socket firewalld.service containerd.service fio-docker-fsck.service compose-apps-register-images
```

Comment:
I don't think it needs to be explicitly called after compose-apps-register-images here. I believe just having the other job with before docker should be enough. Also curious if this could cause a dep chain issue when a condition at compose-apps-register-images causes this job to not be executed.

Reply:
>I don't think it needs to be explicitly called after compose-apps-register-images here.

Yes, you're right, it's not necessary, I put it there because for some reason we put it for another services. I'll remove it.

> Also curious if this could cause a dep chain issue when a condition at compose-apps-register-images causes this job to not be executed.

`After` and `Before` is about ordering of servicing they do not define dependencies as `Requires` does. So, this job (`docker.service`) should be started regardless of `compose-apps-register-images` condition.